src/test/java/org/broadinstitute/consent/http/db/DAOTestHelper.java

```diff
@@ -265,4 +274,23 @@ private DataAccessRequest createDataAccessRequest(Integer userId, Integer collec
     return dataAccessRequestDAO.findByReferenceId(referenceId);
   }
 
+  protected Map<String, FormDataBodyPart> getRandomFiles(Integer count) {
+    Map<String, FormDataBodyPart> mapOfFiles = new HashMap<>();
+    IntStream.range(0, count)
+        .forEach(index -> {
+          String name = String.format("file%d", index);
+          mapOfFiles.put(name, getFormDataBodyPartMock(name));
+        });
+    return mapOfFiles;
+  }
+
+  private FormDataBodyPart getFormDataBodyPartMock(String name) {
+    FormDataBodyPart part = mock(FormDataBodyPart.class);
+    when(part.getName()).thenReturn(name);
+    when(part.getMediaType()).thenReturn(MediaType.MULTIPART_FORM_DATA_TYPE);
+    when(part.getValueAs(InputStream.class))
+        .thenReturn(new ByteArrayInputStream(EMPTY_JSON_DOCUMENT.getBytes()));
+    return part;
+  }
```

Comment:
My preference is to add these as private methods to the test class(es) that use them. What we've found is that this helper class balloons with slight variations of entity creation methods. I'm less concerned about DRY in tests - my experience is that it is usually an anti-pattern for generic data objects. The one thing we do need to share here is the infrastructure setup/teardown that all DAO classes need.